doc/cephadm/troubleshooting.rst

```diff
@@ -231,22 +240,28 @@ Or this error::
 
    Must set public_network config option or specify a CIDR network, ceph addrvec, or plain IP
 
-This means that you must run a command of this form::
+This means that you must run a command of this form:
 
-  ceph config set mon public_network <mon_network>
+.. prompt:: bash
 
-For more detail on operations of this kind, see :ref:`deploy_additional_monitors`
+   ceph config set mon public_network <mon_network>
+
+For more detail on operations of this kind, see
+:ref:`deploy_additional_monitors`.
 
 Accessing the Admin Socket
 --------------------------
 
-Each Ceph daemon provides an admin socket that bypasses the
-MONs (See :ref:`rados-monitoring-using-admin-socket`).
+Each Ceph daemon provides an admin socket that bypasses the MONs (See
```

Comment:
I wouldn't say that the admin sockets _bypass_ mons.  Maybe

Each Ceph daemon (and most clients) can provide an admin socket that can be used to change the running state and query a variety of metrics.

Reply:
please disregard this